### DISCUSSION TITLE
能對摺的筆電才新潮！

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#704] [General]
科技日新月異，現在大多數人手中都有一台筆電，但筆電本身占空間，想帶出門都需要三思甚麼該帶甚麼不該帶，但現在推出了可折疊式的筆電，不占空間、方便收納，比起帶一台平板，一台既擁有多功能又方便收納的折疊式筆電或許更加方便呢!

https://www.coolpc.com.tw/tw/shop/laptop/asus-zenbook-17-fold-oled/

**Comment by AnAnistired** [comment]
筆電本來就能對折😂，但能對折的大螢幕真的超炫砲，雖然耗電量的部分是硬傷🤔

**Comment by dinosaur1009** [comment]
好奇筆電會不會因為可以折所以本身重量也較輕？！感覺這個筆電在空間小的地方也很實用！

**Comment by Yunhans** [comment]
希望能有效能不錯又方便攜帶的筆電出現

**Comment by Chiway1229** [comment]
我有看過別人拿折疊式的觸碰式手機，蠻酷的
有個缺點是摺疊的地方在觸感上會有一種不太舒服的斷層感
不過如果是摺疊筆電就完美避開觸碰的問題了
如果有機會的話，肯定會想要搞一台來玩

**Comment by bonny123865** [comment]
這個筆電看起來還不錯，而且還可以調整模式！
但是現在已經入手筆電跟平板了，瞬間需求降為0
但是真的還蠻新潮的！

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
除了方便外 又有著更舒適的視覺享受  
真的令人很心動

**Comment by evanchou1204** [comment]
感覺只是很炫炮而已，個人是不會想要真的買來用哈哈哈

**Comment by Pietra1226** [comment]
這樣就不用擔心書包裝不下大螢幕的筆電了

**Comment by ghost** [comment]
感覺不錯，如果有可攜帶的大螢幕，在寫程式時就能有更多的版面空間

**Comment by delia07152** [comment]
好多產品的功能性都蠻重疊的，像是筆電跟平板、手機跟平板等等，除了能折疊的噱頭，感覺可以再做一個創新又難以被取代的技術，這樣我可能才會想買。

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
光看就覺的好爽 但價錢的部分就沒那麼親民了 希望哪天我可以變成買東西不看價錢

**Comment by CPCpyt** [comment]
隨然感覺真的很新穎，但我覺得CP值實在有待商議。
個人認為我是不會需要用到這樣的功能。

**Comment by wenchi9023** [comment]
過往人們不斷想要把3C產品的螢幕加大，現在則是希望能折疊、佔據空間越小越好，很有趣的現象!

**Comment by Twenty0522** [comment]
是非常炫砲的設計，之前也很想要買可以觸控的筆電，同時擁有平板的感覺，但這樣的電腦作為平板使用時會覺得可攜性還有方便性較低。而連結所介紹的電腦也有可攜性的問題，我還是比較喜歡傳統的筆電，像是macbook air超級輕便的，走到哪裡拿到哪裡。

**Comment by Sylvie0601** [comment]
我覺得感覺就是單純炫泡ㄟ!我更願意花錢去買配備比較好的筆電，感覺使用上面我還是喜歡一般的筆電，然後再搭配平板這樣。

**Comment by yuxinlin93** [comment]
可以對折的螢幕也太酷了吧 感覺比較像可以對折的大平板 而且有很多不同的使用方式 好方便 ! 不過還要另外接鍵盤 這是需要考慮的點

**Comment by harrylee1971** [comment]
是蠻酷的，但是對於寫code好像有點麻煩，平板的作業系統不知道能否寫程式

**Comment by s96015welson** [comment]
摺疊螢幕真的滿炫砲的，不過產品良率以及耐用程度都還不足，且價格也不親民。不過能看到不論是手機或是筆電用上曲面螢幕，有各種摺法，甚至還有做成卷軸狀的，對消費者也是件好消息，尤其現在的消費性產品外觀同質性太高了，希望之後能多看到如2000年代各種像變形金剛般具有創意的電子產品。

**Comment by a45320614** [comment]
這種新興的科技產品通常都要經過一段時間的測試以及觀察才有一定的穩定性，況且目前的價格真的難以下手，應該不會成為目前筆電市場的主力。就我個人而言，我還是很願意嘗試這種創新的產品的，只是可能要等4、5年產品成熟後再進行購買。

**Comment by HsuChen88** [comment]
三星的摺疊手機有一些獨創的新功能，但這些功能說不上實用，就只是非常特別，等有新的實用的想法出現後，電腦再朝摺疊的方向發展，就有可能會普及化了。

**Comment by 110302550** [comment]
其實我覺得摺疊還是要看人，有人會覺得方便，小小的帶出門，想用的時候打開很節約空間，但有些人追求生活品質，不願意看到螢幕是可折疊式的，看起來感覺怪怪的，我就是這種人，還是無法接受那樣的產品，我喜歡完整性。

**Comment by zephyrxin01** [comment]
我覺得很新穎但同時也會怕使用上會不會比較容易壞，因為像是三星的折疊手機用久了就會容易在折疊處出現痕跡，時間久了美觀上或是觸控的敏銳度可能就會被影響。

**Comment by RogelioKG** [comment]
對折工藝的難點在於鉸鏈的設計和折面螢幕材料的選用。其中鉸鏈的設計也是至關重要的一環（不知道為什麼大家都只關注螢幕材料QQ）。鉸鏈設計常見的有水滴鉸鏈（內折時側看螢幕會捲成水滴狀，因而得名）和外折鉸鏈（我記得外折鉸鏈成本比較高，但效果也沒有內折的好）。

**Comment by ashleyy-huang** [comment]
折疊的技術好像越來越常出現在科技業，透過這個技術就可以提高整個機器的機動性和方便性，但我認為還有很大的進步空間，會擔心這個螢幕的使用耗損和一般螢幕比起來會不會過一下下就要換新的了

**Comment by larry40305** [comment]
可折疊式筆電的出現提供了更方便攜帶和收納的選擇，並滿足了人們對更多功能和靈活性的需求。隨著科技的進步，我們可以期待這類產品的不斷改進和創新。

**Comment by zhiron544** [comment]
我覺得能對摺的筆電是一個很新潮的產品，它可以讓我們更方便地攜帶和使用筆電，也可以適應不同的場合和需求。我想知道這種筆電的性能和價格，以及它有什麼缺點。

**Comment by ThanatosJun** [comment]
當我同時有筆電與平板時，如果接下來的活動需要大量打字，那我必須選擇筆電，這是因為鍵盤的關係。而如果只要看資料等，我會選擇平板，方便又輕。但當對摺筆電出市，我會期許他的未來發展，但還不會嘗試，因為性能的關係，與電競筆電相比應當有段距離，然而之後是否能增加收束的體積，這將是我所期待的。

**Comment by Alex11392** [comment]
我會想，作為一個攜帶式裝置，我真的需要那麼大的螢幕嗎?

**Comment by lin157644** [comment]
我覺得現在折疊式裝置比較可惜的一點，就是螢幕的觸感因為使用塑膠材質而比較不佳，並且耐刮耐劃性也不如玻璃，需要小心呵護，就我了解三星有推出帶有超薄玻璃的機種，但也是僅作為內層增加強度用，希望未來有機會看到手感滑順又耐用的折疊裝置。

**Comment by William08290829** [comment]
這筆電的螢幕可以對折，其目的就是為了讓使用者可以有更大的螢幕但又不佔這麼大的空間。但我自己是比較沒什麼興趣，一是感覺這一摔或是折久了就很容易壞ww，二是好像也不需要這麼大的螢幕，有點太大了ww。

**Comment by wanka2030** [comment]
折疊的科技產品好像會逐年累加，觸控螢幕的演變也讓現在的手機、平板以及電腦都日新月異，除了內部軟體有著人工智慧的輔助之外，外部的硬體也需要有不斷的更進，例如三角形的摺疊手機、華碩的摺疊筆電，以後假如有什麼折疊機車應該也不意外了😳

**Reply by jeff110303549** [reply]
可折疊式筆電確實是一個很棒的創新！它結合了筆電和平板的功能，同時又具有方便攜帶和收納的優勢。這種設計讓人不再需要為了帶筆電而煩惱，同時也滿足了人們對於便攜性和多功能性的需求。隨著科技的不斷進步，我們可以期待更多這樣的創新產品出現，或許之後的平板將不再需要一個實體、能夠透過擴增實境的互動完成更多操作也不一定！

**Comment by Piyenparty** [comment]
我也有注意到那種折疊式觸控手機，真的蠻有未來感的，像是把平板縮進口袋裡一樣。看起來很帥，但你說的那個「中間摺痕的觸感斷層」真的不少人反映過，滑過去會有一種凹下去的觸感，對追求螢幕平滑體驗的人來說確實有點出戲。

但如果應用在筆電上就聰明多了！因為筆電本來就是「打開使用、闔起收納」的設計邏輯，而且操作上多半是用鍵盤跟觸控板，不會一直用手滑螢幕，自然也就避開了那條折痕的干擾。

要是真的出現又輕又耐用、而且摺疊後還超薄的筆電，我相信絕對會讓人想搞一台來玩玩看，光是想到能隨時把大螢幕帶在身上、又不用背一大塊就覺得超方便。如果價格再親民一點，那應該會變成新一代的夢幻機種了！

**Comment by RU10171017** [comment]
看到這篇我也蠻有感的，最近越來越多摺疊式筆電的廣告開始出現，像 ASUS 那台 17 吋可以對摺的 OLED 筆電，一開始看到真的蠻驚艷的，覺得科技終於讓筆電變得「輕薄但不犧牲螢幕大小」，帶出門又不佔太多空間，這點真的很吸引人。不過說實話，我自己對這類產品還是有一點觀望。一方面是價格真的不便宜，再來就是摺疊螢幕的耐用度和實用性，還不太確定能不能應付日常敲打與多工需求。畢竟筆電不像手機，使用時間跟頻率更高，一但螢幕出問題維修應該也不便宜QQ

**Comment by wuyu06** [comment]
摺疊筆電真的很吸引人，尤其對常常外出或空間有限的人來說，方便攜帶是大優點。不過這類產品還在發展階段，價格和耐用度可能還有待觀察。未來如果技術成熟又實用，摺疊筆電很可能成為潮流新標準，改變我們工作和生活的方式

**Comment by mattwang1222** [comment]
我覺得可折疊筆電確實讓移動工作更自由，尤其對常出門的人來說非常有吸引力。不過目前價格偏高、耐用度也還需要觀察，真正普及前可能還是得靠技術成熟與價格下修。

**Comment by edawdewv** [comment]
折疊式觸控手機確實帶來了很有趣的科技體驗，尤其是那種能讓大螢幕變得便攜的設計，讓人感覺未來感十足。不過，中間的摺痕問題一直讓使用者感到困擾，滑動時會有明顯的凹陷感，這對習慣流暢操作的人來說確實是個缺點。
但若把這種摺疊技術應用到筆電上，情況可能會完全不同。筆電本身就設計成開合使用，主要的操作是鍵盤和觸控板，不像手機頻繁用手指滑動螢幕，因此摺痕帶來的不適感會大大減少。
我想未來若能推出既輕便又耐用的摺疊筆電，不僅能帶來攜帶便利，也能滿足對大螢幕需求的使用者。再加上價格能夠更親民，這種筆電極有可能成為市場上的新潮流，甚至改變我們對筆電便攜性的期待。
